install_nvm.sh

```diff
@@ -0,0 +1,425 @@
+#!/usr/bin/env bash
```

Comment:
File should not be committed as part of project. Will need to remove this file from git

Reply:
`git rm --cached install_nvm.sh`